bugsnag/src/main/java/com/bugsnag/Configuration.java

```diff
@@ -35,7 +35,7 @@ public class Configuration {
     public String appType;
     public String appVersion;
     public Delivery delivery;
-    public EndpointConfiguration endpointConfiguration;
+    public EndpointConfiguration endpoints;
```

Comment:
Given that we already have `setEndpoints` methods in the `Configuration` class, maybe we should also make the field `private` (we should probably add a `getEndpoints()` as well in that case)?

```suggestion
    private EndpointConfiguration endpoints;
```

Reply:
i'm just about to do the encapsulation task and will make sure to get these accessors right there 👍